### ISSUE TITLE
[Zone] [Hellfire Pensiula] 

### ISSUE THREAD

**Issue by Nadiirr** [SunwellWoW/Sunwell-TBC-Bugtracker#115]
- [x] Arzeth the Merciless should have higher melee damage, atleast 50% higher than his Arzeth the Powerless version, 2 minute respawn time (after either Arzeth the Merciless or Arzeth the Powerless is killed)

- [x] Fel Soldier should have higher melee damage, comments from TBC era mention them hitting for 1500 on and around 1750 with Cleave so their base damage before armor reduction from players is prob something around 2,190 - 2,655, they should also use Cleave (Spell ID 15496) every 6 seconds or so and Cut Down (Spell ID 32009) which deals weapon damage and knocks back. Should be noted that they currently have 0 armor, Felguard Legionnaire in Nagrand are weaker than them and have 6,042 armor for comparison. Should be immune most most CCs such as Fear/Stun/Root and should be linked together with Wrath Master and be part of event where Burning Legion is attacking the defenders of Dark Portal.
- [x] Wrath Master should have higher melee damage around, base damage before armor reduction from players/mobs should be 6,290 - 7,470. They should be immune to any kind of CC such as Fear/Stun/Root etc. Fel Soldier and Wrath Master should be linked together and be part of event where Burning Legion is attacking The Dark Portal. Periodically Wrath Master and 4 Fel Soldiers should spawn at Gateway Shaadraz or Gateway Murketh and then slowly walk the path towards Pit Commander. Wrath Master should be in middle and slightly ahead while Fel Soldiers move in same manner alongside him 2 on each side, when they reach Pit Commander they should engage the defenders of The Dark Portal in combat.
- [x] Dread Tactician should have 28 000 hp and use spells such as Carrion Swarm (Spell ID 33678), Sleep (Spell ID 12098) and Inferno, they should be enslavable by warlocks and have their spells usable while enslaved aswell. Currently they have 20 armor which should be around 6k and hit very weak, base damage before reductions should be in 1,030 - 1,178 range.
- [x] Deathwhisperer missing Dual Wield ability, base damage before reductions should in 1,802 - 2,186 range. Should use Mind Flay (Spell ID 32417) and Thrash, if she crits she should Flurry (Spell ID 15088). Enslavable by warlock.
- [x] Felguard Destroyer should be moved to correct locations which are (74, 39/74. 38.4) ; (71.6, 45.6/70.8, 45.2/70.6, 45.6) ; (73.2, 57.2/72.8, 58.6) ; (72.0, 55.6/71.6, 54.8/71.4, 55.2)  basicly 2 Felguard Destroyers on each side (4 in total) of Path of Glory , they should use Sweeping Charge (Spell ID 33971) which hits for 400, Cleave (Spell ID 40505) on a 6 second cooldown and they should use Mortal Strike (Spell ID 40220). Their base damage before reduction should be increased to something around 500-600 range.
- [x] Pit Commander should use Rain of Fire that ticks for 60000 (Spell ID 33972), Cleave which hits for 150% of his normal damage (Spell ID 30619) and his normal melee before reductions should be in 8,908 - 10,575 range. About 5 minutes 30 seconds after death he should respawn at gateway Murketh or Shaadraz and slowly walks back to his original position and resume his duty.
- [x] Infernal Siegebreaker base damage before reductions should be in 3,712 - 4,408 range. Every minute or so Pit Commander casts a spell and summons these on specific spots to engage the defenders of The Dark Portal.

- [x] Fel Reaver should use his War Stomp which hits for around 2-3k on medium armor (Spell ID 41534), since his War Stomp is normal damage + 500 his melee damage needs to be adjusted aswell.  Every time Fel Reaver roars/shakes ground he un-stealths rogues and druids in the area around him even if they are outside his aggro range.

In case developers want to revert Throne of Kil'jaeden area back to pre-2.4 i will post information about mobs that were present there.
- [ ] Deathforge Automaton 78,000 - 81,000 health (71 - 72 level), base damage unknown but should be the highest out of all mobs there not counting Kazzak and armor should be in same category as other Fel Reavers at lvl 70. They have Sonic Burst (Spell ID 23918), Earthquake (Spell ID 19798) and some kind of knockback passive to those who attack him in melee (its prob Spell ID 47113). Should be noted that you can even complete quest for Shattered Halls key on their corpses too. When in idle position they should be in "hunch" animation. This video shows where their original position was  https://www.youtube.com/watch?v=jf51zsgl-FA&feature=youtu.be  
- [ ] Throne Hound 62,000 - 65,000 health (70 - 71 level), Immune to enslave and banish, goes into a frenzy frequently (Spell ID 31540 or 23128?), deals around 2k damage on 3955 armor (video below)
- [ ] Throne Guard Sentinel 63,000 - 65,000 health (71 - 72 level), they have Impending Doom which deals 5k damage after 30 seconds (Spell ID 39046) and also has a 4 second stun War Stomp on a low cooldown. These should be positioned where Felblood Initiates are now and channel some green visual aswell.
- [ ] Throne-Guard Champion 63,000 health (72 level), they have Rain of Fire and an AoE Fear, they patrol to certain areas on the Throne of Kil'jaeden.
- [ ] Throne-Guard Highlord 65,000 health (72 level), insanely high movement speed and has charge type ability that stuns + deals damage and drops all aggro (he will randomly drop aggro and charge people) and Uppercut which deals damage and knockbacks, he patrols entire Throne of Kil'jaeden area and seems to be most dangerous there due to his damage and the combination of his abilities which can knock people to their death from the Throne of Kil'jaeden area if someone is unfortunate enough to stand at the edge.

https://www.youtube.com/watch?v=Ln7zdg4jcLk Shows position of some of the mobs in the area and the events like Throne-Guard Champion casting something into Fel Spark, Throne Hounds damage on 3955 armor, after 4:12 you can see positioning of some of the mobs there.

https://www.youtube.com/watch?v=BvQ2ZPO9JkE Shows position of some of the mobs in the area aswell and events like Throne-Guard Champion casting something into one of the buildings/doodads there.

Quests: 
- [x] Crimson Crystal Clue event doesnt play after you turn it in.
- [x] Natural Remedies Goliathon doesnt spawn after the quest item is used.
- [x] A Spirit Guide when you use the item you summon Ancestral Spirit Wolf NPC but the leveling up sound and animation is heard and seen (this should be hidden), health of Ancestral Spirit Wolf seems off because he has only 3398 if you are doing this at lvl 70 (at lvl 60 his hp is prob really low) and most importantly the Ancestral Spirit Wolf goes under the mountain instead of moving on his proper path.
- [x] NPC - Tunneler - Mob stays underground while aggro'd. Mob should attack you.

~~ **DONE** ~~

- [x] Missing brazier outside of Blood Furnace!!!

![Image](https://i.gyazo.com/212f9786c86f8c5e997d7066cec3a8c6.png)
- [x] Wrath Herald not attacking with offhand (missing Dual Wield ability), base damage before reductions should be in 559 - 670 range, despite having mana they shouldnt cast anything (thats how it was on retail atleast)
- [x] Incandescent Fel Spark should be immune to Fire damage.
- [x] Felblood Initiate should use Sinister Strike (Spell ID 14873) and Bitter Withdrawal (Spell ID 29098)
- [x] Raging Colossus they should spawn adds called Raging Shardling at 75%, 50%, and 25%. As they spawn the adds they shrink in size and power hitting for less, below 25% they should Enrage (Spell ID 18501), Raging Shardlings also have same enrage but its used at around 50% hp.
- [x] Goliathon should use Trample and spawn adds in same manner as Raging Colossus at 75/50/25% hp.
- [x] Shattered Hand Warlock doesnt cast Immolate at all. Tested these mobs again and found out they do cast it but its done very very rarely, should be changed so they cast it at start of fight then spam Shadowbolt and recast it again when it wears off on enemy.
- [x] Drillmaster Zurok interrupts his Whirlwind often especially if you move, he should stay in place during channel.
- [x] Force-Commander Gorax uses his Charge but he doesnt move to charged target, he simply stuns it from range and nothing else.
- [x] Hand of Kargath same issue as Force-Commander Gorax with Charge, also he should spawn with 2 Shattered Hand Berserkers when you plant the flag, right now he spawns alone.
- [x] Shattered Hand Acolyte when he tries casting Mind Flay it interrupts (i think this is same for other mobs with channeling spells), also he doesnt cast Heal when he drops below 50%.
- [x] Shattered Hand Berserker doesnt have Dual Wield ability, they only attack with main hand while they should be attacking with both (offhand deals 50% damage), also their Enrage seems to be random because when i was fighting them one of them enraged at 5% other one at 25% and third one at 10%.
- [ ] Warboss Nekrogg doesnt use Bloodthirst or Bonechewer Bite and has same issue as Shattered Hand Berserkers with Dual Wield ability (he only attacks with main hand instead of attacking with both)
- [x] Hulking Helboar uses Hulk at 100% only once and never again. They should use Hulk at 66% of their hp and once again at 33% of their hp, Hulk stacks so first time they "Hulk" they should do 10% more damage and have bigger size and second time they "Hulk" they should do 20% more damage and once again bigger size. Every time they "Hulk" there should be an emote saying "Hulking Helboar begins to grow stronger." Also they should have something like a thorns passive ability that deals 20-30 damage to melee attackers when struck, i think this ability is called Burning Spikes because i recall other Helboars having same thing.
- [x] Starving Helboar, they should have Burning Spikes same as above and they should Charge when aggroed, also a comment on wowhead from TBC era noted that "These guys have an unnaturally huge aggro range. It's about 3x bigger than normal mobs."
- [x] Bonechewer Evoker they are casting Fireball properly but missing their Fire Shield/Immolation passive, comment on wowhead noted that if you go into their melee range you will start taking fire damage quickly.
- [x] Bonechewer Raider this guy should dismount when aggroed and summon his wolf but right now he stays on mount and summons his wolf to attack you (when wolf is summoned his level up sound and animation is seen, this should be hidden)
- [x] Tagar Spinebreaker doesnt have his weapon (he should use a weapon that looks like Arcanite Reaper), also he doesnt use his abilties such as Spine Break (8 second stun) and Whirlwind Knockback (instant ability that deals AOE weapon damage + knockback)
- [x] Infernal Warbringer missing Immolation passive which deals fire damage to units in melee range and they should be immune to fire damage.
- [x] Fel Cannon MKI when aggroed they just stand and do nothing, if you go into melee range they will start meleeing you for some small damage. They shouldnt melee at all but spam ability called Fel Cannon Blast until they die.
- [x] Razorsaw should use abilities called Saw Blade and Slime Spray.
- [x] Mo'arg Forgefiend should use same abilities as Razorsaw (Saw Blade and Slime Spray)
- [x] Sister of Grief should use Lash of Pain quite often (cooldown is around 6 seconds) and cast Whipped Fury on nearby ally (Spell ID 34250)
- [x] Forge Camp Legionnaire should use Cleave and Mortal Strike (Cleave ID 15496 110% weapon damage, Mortal Strike ID 32736 100% weapon damage + 50% healing debuff)
- [x] Warbringer Arix'amal doesnt attack with both weapons like he should (missing Dual Wield ability) and doesnt use any abilities. He should use Cutdown (Spell ID 32009), Cleave (Spell ID 15496, it is also noted he uses his abilities more often than Legionnaire in the area so he is more dangerous), Fel Fire (Spell ID 32008, because he is dual wielding this would add quite nice damage to him) and Flame Wave (Spell ID 33803).
- [x] Mistress of Doom should summon imps on aggro (Spell ID 34237) and should have Dual Wield (currently she only attacks with main hand)
- [x] Arazzius the Cruel should be immune to Fear, Curse of Exhaustion and any other CC such as stuns,slows, roots etc also he is missing his Inferno ability which is an AOE stun that deals minor damage and summons 2 Infernals.
- [x] Vorakem Doomspeaker doesnt attack with his offhand (missing Dual Wield ability) and every time he uses his Flame Wave ability the leveling up sound is heard and animation is seen on the invisible infernal dummy from that spell (this should be hidden)
- [x] Doomwhisperer should be attacking with offhand aswell (missing Dual Wield ability)
- [x] Subjugator Shi'aziv should be attacking with offhand aswell (missing Dual Wield ability) and she should use Mind Sear and Pain Spike.
- [x] Subjugator Yalqiz should be attacking with offhand aswell (missing Dual Wield ability) and she should use Mind Sear and Pain Spike.
- [x] Fel Handler should be linked together with Netherhound, they patrol together and once one of them is aggroed the other one is pulled aswell.
- [x] Wrathguard doesnt attack with his offhand (missing Dual Wield ability) also he is missing his Flame Wave ability
- [x] Maiden of Pain doesnt use Lash of Pain on main target and Whipped Frenzy on other nearby friendly units.
- [x] Gan'arg Sapper isnt casting Set Charge (Spell ID 33895) and Power Burn (Spell ID 33974), Set Charge should be used randomly during fight while Power Burn when they are at low health.
- [x] Urga'zz isnt using any of his abilities, should use Drill (Spell ID 32004), Throw Proximity Bomb (Spell ID 34095) and Power Burn (Spell ID 32003) when he is at low health.
- [x] Z'kral isnt using any of his abilities, should use Drill (Spell ID 32004), Throw Proximity Bomb (Spell ID 34095) and Power Burn (Spell ID 32003) when he is at low health.
- [x] Vacillating Voidcaller should have ability called "Elemental Response" which works like this: When they get hit by by a specific school of magic they cast a self buff that reduces damage taken from that school of magic by 75% and they start spamming a spell depending on what school of magic hit them, if its Shadow they will start using Shadow Bolt, if its Fire then Fireball, if its Frost they then Frostbolt, if its Nature then Lightning Bolt, if its Holy then Holy Bolt, if its Arcane then Arcane Missiles (all those spells cast with the exception of Arcane Missiles have 2 second base cast time), if no school of magic is used then they will just use Shadow Bolt with no self buff.
- [x] Void Baron Galaxis should have Absorb Life ability, comments mention that this is not actually casted on enemy but something like a self buff, if you start damaging him instead of killing the adds first he will kill off one of the add and then get a 5000 heal over 5 seconds and repeat the same if other add is alive when his health gets lower, if you kill adds first then start damaging him he cannot heal.
- [x] Terrorfiend every time he tries casting his Fel Flames it gets interrupted, he uses War Stomp properly.
- [x] Bleeding Hollow Tormentor should use Fear.
- [x] Bleeding Hollow Necrolyte should start with a Bleeding Hollow Skeleton, if there is a nearby corpse they would use Raise Dead (Spell ID 34019) to spawn another one to aid them in combat.
- [x] [ ] Bleeding Hollow Dark Shaman should cast Bloodlust on himself or ally after 10 seconds or so after aggroing, also should cast Raise Soul if there is a corpse nearby which summons a Bleeding Hollow Soul mob to assist him in combat.
- [x] Worg Master Kruush when you aggro him he should use ability called "Fear the Worgs" which basicly calls his wolves to assist him in combat against you, when you aggro him the emote "Worg Master Kruush calls for his worgs" is there but the effect isnt.
- [x] Unyielding Knight They are casting Heal Other but the effect isnt there.
- [x] Unyielding Sorcerer They should cast Flamestrike quite often and should have increased aggro radius compared to other mobs in the Expedition Armory.
- [x] Arch Mage Xintor when he uses Arcane Missiles it gets interrupted (same issue with other mobs who have channeled spells), when engaged he should cast Ice Barrier upon himself then chain cast Fireball, Frostbolt and Arcane Missiles until he runs out of mana.
- [x] Rogue Voidwalker shouldnt be holding an axe/hatchet in main hand.
- [x] Marauding Crust Burster should use Bore when his aggro target is in melee range.
- [x] Stonescythe Alpha when their health drops to 50% or lower they should use ability called "Desperate Defense" with "defend themselves in desperation" emote.
- [x] Stonescythe Whelp should use same "Desperate Defense" ability as Alpha does at 50% hp.
- [x] Haal'eshi Talonguard should use Sprint (Spell ID 32720) if enemy is engaging from from range and if enemy tries to run away during the fight (comments mention they tried kiting them and its impossible because they will spam it to catch up)
- [x] Haal'eshi Windwalker should use Lightning Bolt and Hurricane.
- [x] Avruu should use Judgement of Darkness constantly 100% uptime (Spell ID 34112) and Dark Mending (when he gets below 50% hp)
- [x] Aeranas should be immune to Nature damage.
- [x] Illidari Taskmaster should constantly keep Demoralizing Shout on its enemy (100% uptime), use Hamstring and should be linked to Dreghood Brutes, during fight he should cast Haste Other on Brutes, when Taskmaster is killed the Dreghood Brutes/Geomancers should turn friendly and change to Fleeing Dreghood Warrior/Geomancer, run away and despawn. Also on aggro he should yell either "Slay these intruders, filthy Dreghood! I promise you neverending pain if you disobey me!" or "Go you little wretches! Show these fools that our master is not to be trifled with!"  When not engaged in combat he will sometimes yell "Do not stop! I promise a thousand deaths if you even think of putting down that pick!" also when he uses Haste Other during combat.
- [x] Dreghood Brute they should be linked to Illidari Taskmaster and when he is pulled they should say either "Forgive me... I have no choice." or  "I will do as you ask, demon... at least for now."
- [x] Dreghood Geomancer they should be linked to Illdari Taskmaster and when he is pulled they should either "Forgive me... I have no choice." or  "I will do as you ask, demon... at least for now."

- [x] Quest "The Finest Down" Object [Kaliri Nest] Click it and 1-2 Kaliri should spawn; sometimes kaliri hatchlings, sometimes kaliri matriarchs. Currently does nothing.

~~NPC - Bleeding Hallow Dark Shaman - Does not chase you. Stays stationary inside building even when you are LOS~~

- [x] NPC - Wounded Blood Elf Pilgrim - She is in a cage but will aggro out of the cage when nearby.

- [x] NPC - Fulgorge - Should move underground like the tunneler.

 ITEM - Cleansing Vial - for quest [Cleansing the Waters] - when casting the item it says "Item Cleansing Vial DND" Not sure what it should say but "Cleansing Vial" would be better. _[client side thing,can't override spell names]_

- [x] NPC - Aggonis - Should have [fear] ability.

**Comment by Silroth** [comment]
NPCs: 
- Shattered Hand Warlock doesnt cast Immolate at all.
- Drillmaster Zurok interrupts his Whirlwind often especially if you move, he should stay in place during channel.
- Force-Commander Gorax uses his Charge but he doesnt move to charged target, he simply stuns it from range and nothing else.
- Hand of Kargath same issue as Force-Commander Gorax with Charge, also he should spawn with 2 Shattered Hand Berserkers when you plant the flag, right now he spawns alone.
- Shattered Hand Acolyte when he tries casting Mind Flay it interrupts (i think this is same for other mobs with channeling spells), also he doesnt cast Heal when he drops below 50%.
- Shattered Hand Berserker doesnt have Dual Wield ability, they only attack with main hand while they should be attacking with both (offhand deals 50% damage), also their Enrage seems to be random because when i was fighting them one of them enraged at 5% other one at 25% and third one at 10%.
- Warboss Nekrogg doesnt use Bloodthirst or Bonechewer Bite and has same issue as Shattered Hand Berserkers with Dual Wield ability (he only attacks with main hand instead of attacking with both)
- Hulking Helboar uses Hulk at 100% only once and never again. They should use Hulk at 66% of their hp and once again at 33% of their hp, Hulk stacks so first time they "Hulk" they should do 10% more damage and have bigger size and second time they "Hulk" they should do 20% more damage and once again bigger size. Every time they "Hulk" there should be an emote saying "Hulking Helboar begins to grow stronger." Also they should have something like a thorns passive ability that deals 20-30 damage to melee attackers when struck, i think this ability is called Burning Spikes because i recall other Helboars having same thing.
- Starving Helboar, they should have Burning Spikes same as above and they should Charge when aggroed, also a comment on wowhead from TBC era noted that "These guys have an unnaturally huge aggro range. It's about 3x bigger than normal mobs."
- Bonechewer Evoker they are casting Fireball properly but missing their Fire Shield/Immolation passive, comment on wowhead noted that if you go into their melee range you will start taking fire damage quickly.
- Bonechewer Raider this guy should dismount when aggroed and summon his wolf but right now he stays on mount and summons his wolf to attack you (when wolf is summoned his level up sound and animation is seen, this should be hidden)
- Tagar Spinebreaker doesnt have his weapon (he should use a weapon that looks like Arcanite Reaper), also he doesnt use his abilties such as Spine Break (8 second stun) and Whirlwind Knockback (instant ability that deals AOE weapon damage + knockback)
- Infernal Warbringer missing Immolation passive which deals fire damage to units in melee range and they should be immune to fire damage.
- Fel Cannon MKI when aggroed they just stand and do nothing, if you go into melee range they will start meleeing you for some small damage. They shouldnt melee at all but spam ability called Fel Cannon Blast until they die.
- Razorsaw should use abilities called Saw Blade and Slime Spray.
- Mo'arg Forgefiend should use same abilities as Razorsaw (Saw Blade and Slime Spray)
- Sister of Grief should use Lash of Pain quite often (cooldown is around 6 seconds)
- Forge Camp Legionnaire should use Cleave and Mortal Strike (Cleave ID 15496 110% weapon damage, Mortal Strike ID 32736 100% weapon damage + 50% healing debuff)
- Warbringer Arix'amal doesnt attack with both weapons like he should (missing Dual Wield ability) and doesnt use any abilities. He should use Cutdown (Spell ID 32009), Cleave (Spell ID 15496, it is also noted he uses his abilities more often than Legionnaire in the area so he is more dangerous), Fel Fire (Spell ID 32008, because he is dual wielding this would add quite nice damage to him) and Flame Wave (Spell ID 33803).
- Mistress of Doom should summon imps on aggro (Spell ID 34237) and should have Dual Wield (currently she only attacks with main hand)
- Arazzius the Cruel should be immune to Fear, Curse of Exhaustion and any other CC such as stuns,slows, roots etc also he is missing his Inferno ability which is an AOE stun that deals minor damage and summons 2 Infernals.
- Vorakem Doomspeaker doesnt attack with his offhand (missing Dual Wield ability) and every time he uses his Flame Wave ability the leveling up sound is heard and animation is seen on the invisible infernal dummy from that spell (this should be hidden)
- Doomwhisperer should be attacking with offhand aswell (missing Dual Wield ability)
- Subjugator Shi'aziv should be attacking with offhand aswell (missing Dual Wield ability) and she should use Mind Sear and Pain Spike.
- Subjugator Yalqiz should be attacking with offhand aswell (missing Dual Wield ability) and she should use Mind Sear and Pain Spike.
- Fel Handler should be linked together with Netherhound, they patrol together and once one of them is aggroed the other one is pulled aswell.
- Wrathguard doesnt attack with his offhand (missing Dual Wield ability) also he is missing his Flame Wave ability
- Vacillating Voidcaller should have ability called "Elemental Response" which works like this: When they get hit by by a specific school of magic they cast a self buff that reduces damage taken from that school of magic by 75% and they start spamming a spell depending on what school of magic hit them, if its Shadow they will start using Shadow Bolt, if its Fire then Fireball, if its Frost they then Frostbolt, if its Nature then Lightning Bolt, if its Holy then Holy Bolt, if its Arcane then Arcane Missiles (all those spells cast with the exception of Arcane Missiles have 2 second base cast time), if no school of magic is used then they will just use Shadow Bolt with no self buff.
- Void Baron Galaxis should have Absorb Life ability, comments mention that this is not actually casted on enemy but something like a self buff, if you start damaging him instead of killing the adds first he will kill off one of the add and then get a 5000 heal over 5 seconds and repeat the same if other add is alive when his health gets lower, if you kill adds first then start damaging him he cannot heal.
- Terrorfiend every time he tries casting his Fel Flames it gets interrupted, he uses War Stomp properly.
- Bleeding Hollow Tormentor should use Fear.
- Bleeding Hollow Necrolyte should start with a Bleeding Hollow Skeleton, if there is a nearby corpse they would use Raise Dead (Spell ID 34019) to spawn another one to aid them in combat.
- Bleeding Hollow Dark Shaman should cast Bloodlust on himself or ally after 10 seconds or so after aggroing, also should cast Raise Soul if there is a corpse nearby which summons a Bleeding Hollow Soul mob to assist him in combat.
- Worg Master Kruush when you aggro him he should use ability called "Fear the Worgs" which basicly calls his wolves to assist him in combat against you, when you aggro him the emote "Worg Master Kruush calls for his worgs" is there but the effect isnt.
- Unyielding Knight They are casting Heal Other but the effect isnt there.
- Unyielding Sorcerer They should cast Flamestrike quite often and should have increased aggro radius compared to other mobs in the Expedition Armory.
- Arch Mage Xintor when he uses Arcane Missiles it gets interrupted (same issue with other mobs who have channeled spells), when engaged he should cast Ice Barrier upon himself then chain cast Fireball, Frostbolt and Arcane Missiles until he runs out of mana.

Will add more info on other half of Hellfire Peninsula mobs tomorrow or the day after.

**Comment by notsofamous** [comment]
Ancestral Spirit Wolf (17077) should be friendly to alliance and horde, currently hostile with both.

**Comment by notsofamous** [comment]
- [ ] Missing brazier outside of Blood Furnace!!!

![Image](https://i.gyazo.com/212f9786c86f8c5e997d7066cec3a8c6.png)

**Comment by Dootz** [comment]
Almost all issues resolved, PTR not updated yet. Thank you for testing and reporting. Throne of Thunder related points moved to separate issue.
